sql/npc_list.sql

```diff
@@ -9134,8 +9134,8 @@ INSERT INTO `npc_list` VALUES (17175333,'Moogle','Moogle',0,0.000,0.000,0.000,0,
 INSERT INTO `npc_list` VALUES (17175334,'Moogle','Moogle',0,0.000,0.000,0.000,0,40,40,0,0,0,2,4194307,0x0000520000000000000000000000000000000000,0,'WOTG');
 INSERT INTO `npc_list` VALUES (17175335,'Moogle','Moogle',0,0.000,0.000,0.000,0,40,40,0,0,0,2,4194307,0x0000520000000000000000000000000000000000,0,'WOTG');
 INSERT INTO `npc_list` VALUES (17175336,'Mithranbivouac','Moogle',203,386.661,-16.351,58.756,32769,40,40,0,16,112,0,3,0x0000340000000000000000000000000000000000,2,'WOTG');
-INSERT INTO `npc_list` VALUES (17175337,'Indescript_Markings','Mithran Bivouac',228,-358.657,-8.012,98.512,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'WOTG');
-INSERT INTO `npc_list` VALUES (17175338,'16','Indescript Markings',0,0.000,0.000,0.000,0,40,40,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,'WOTG');
+INSERT INTO `npc_list` VALUES (17175337,'Mithran_Bivouac','Mithran Bivouac',228,-358.657,-8.012,98.512,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'WOTG');
```

Comment:
okay, thanks @starlightknight. I tried using a mention in my earlier comment, but it didn't autocomplete, then I checked your commits in the darkstar-utils repo and they were unlinked to a github account /shrug